pkg/network/tracer/tracer_windows.go

```diff
@@ -309,6 +309,11 @@ func (t *Tracer) DebugDumpProcessCache(_ context.Context) (interface{}, error) {
 	return nil, ebpf.ErrNotImplemented
 }
 
+// GetNetworkID is not implemented on this OS for Tracer
+func (t *Tracer) GetNetworkID() (string, error) {
+	return "", ebpf.ErrNotImplemented
```

Comment:
Any reason to not use `id, err = ec2.GetNetworkID()` here as well? You can run the tracer on windows

Reply:
I will add an impl here

Reply:
after talking to WKIT: the windows tracer is not supported running in a container, so we don't need this